src/analysis/base.d

```diff
@@ -56,7 +56,7 @@ public:
 
 protected:
 
-	bool inAggregate = false;
+	bool inAggregate;
```

Comment:
For sure there's one. @Hackerpilot uses tabs on his stuff. Excepted libdparse because it was planned to be included in phobos.